Makefile

```diff
@@ -65,6 +65,7 @@ endif
 
 define load_launchd
 	# Hint: try `launchctl enable system/$(1)` if the `launchctl bootstrap` command below fails
+	logger "Installing socket_vmnet version $$(git describe --tags)"
```

Comment:
Do you want to add `logger` for  the `install` and `uninstall` targets too?

Reply:
Sure, will add.